packages/svelte-form/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@tanstack/svelte-form",
+  "version": "1.0.0",
```

Comment:
marked this as 1.0 but if you wanna play it safe you could do `0.5.0` or sth first to see if there are any glaring mistakes made that require a breaking change

Reply:
I think our TanStack Config currently forces all packages in the repo to have the same version, so it will release as v1.x.0 when released